better-control-over-production-changes/README.md

```diff
@@ -0,0 +1,20 @@
+# Better control over changes in production environments
+
+We currently use `flux` to deploy changes to our Management Clusters. `flux` is reconciling the `main` branch on different repositories.
+We have some customers that don't want all changes to be deployed continuously to our production installations.
+Instead, they want us to deploy to a testing installation first, and then deploy to the production installation during a maintenance window.
+
+## Current solution
+
+In order to do that, we keep `flux` suspended on this production installation while it's running on the testing installations. Then, we we want to upgrade the production installation, we manually resume `flux`, wait for the changes to be applied and suspend `flux` again.
+
+## Proposed alternative
+
+We create a new branch called `production` in the `capa-app-collection` and `config` repositories. And we make `flux` running in our production installation to target and reconcile this `production` branch. 
```

Comment:
I guess this way in the future we could also manage more than 1 channel? Like maybe a user-facing RC/staging channel?

Reply:
Also cc @piontec as it relates to Honeybadger in two ways:
- Honeybadger is planning to work on MCF improvements that would enable us more flexibility with installations
- I'm not sure app collections are something we want to keep in general /cc @alex-dabija

Reply:
Sorry @puja108 , I meant to close this RFC but I forgot. I opened the RFC without knowing [this existed](https://github.com/giantswarm/giantswarm/issues/26519), so I'm going to add my feedback there instead. I believe the way forward is moving apps into MCF instead of using the collections.